docs/index.asciidoc

```diff
@@ -165,6 +166,14 @@ actual hostname of the connecting client.
 
 The fully-qualified email address for the From: field in the email.
 
+[id="plugins-{type}s-{plugin}-headers"]
+===== `headers` 
```

Comment:
It is defined as `custom_headers` in the code

Reply:
thanks for spotting!